tests/space-member-test.ts

```diff
@@ -0,0 +1,86 @@
+import { Relay, run } from "../main.ts";
+import { InMemoryAccountContext } from "../nostr.ts/nostr.ts";
+import { fail } from "https://deno.land/std@0.224.0/assert/fail.ts";
+import { assertEquals } from "https://deno.land/std@0.220.1/assert/assert_equals.ts";
+import { prepareSpaceMember } from "../nostr.ts/space-member.ts";
+
+const test_ctx = InMemoryAccountContext.Generate();
+const test_kv = async () => {
+    try {
+        await Deno.remove("test.sqlite");
+    } catch (e) {}
+    return await Deno.openKv("test.sqlite");
+};
+
+Deno.test("Space Member", async (t) => {
+    const relay = await run({
+        auth_required: false,
+        admin: test_ctx.publicKey,
+        default_policy: {
+            allowed_kinds: "none",
+        },
+        kv: await test_kv(),
+    }) as Relay;
+    const new_member = InMemoryAccountContext.Generate();
+
+    await t.step("admin can add member", async () => {
+        const add_member_event = await prepareSpaceMember(test_ctx, new_member.publicKey.hex);
+        if (add_member_event instanceof Error) fail(add_member_event.message);
+        const r = await fetch(`${relay.http_url}`, {
+            method: "POST",
+            body: JSON.stringify(add_member_event),
+        });
```

Comment:
Let's define a function in `nostr.ts` to do the same kind of work.

Reply:
I can do this when I work as a client tomorrow.